### PR TITLE
Fix jenkins tests (pass 1)

### PR DESCRIPTION
**Pull request by JohnLCaron**
This change is [Reviewable](https://reviewable.io/reviews/unidata/netcdf-java/872)

### CODE REVIEW

**Comment by JohnLCaron** [comment]
ready

File: grib/src/test/java/ucar/nc2/grib/grid/TestGribGridTimeCS.java [hunk 1]
```diff
@@ -296,8 +298,19 @@ public void testOffsetIrregular(String endpoint, String gridName, int[] expected
         assertThat(times.size()).isLessThan(ntimes + 1);
         int offsetIdx = 0;
         for (CalendarDate time : times) {
-          CalendarDate expected = baseForRun.add((long) offset.getCoordDouble(offsetIdx++), offsetPeriod);
-          assertThat(time).isEqualTo(expected);
+          double val = offset.getCoordDouble(offsetIdx);
+          // this only works if val is integral
+          if (Misc.nearlyEquals(val, (long) val, Misc.defaultDiffFLoat)) {
+            CalendarDate expected = baseForRun.add((long) offset.getCoordDouble(offsetIdx), offsetPeriod);
+            if (!time.equals(expected)) {
```
Comment (haileyajohnson) [inline grib/src/test/java/ucar/nc2/grib/grid/TestGribGridTimeCS.java:305]: Was this supposed to stay in or is this for debugging?

**Review by haileyajohnson — APPROVED** [review]

File: grib/src/test/java/ucar/nc2/grib/grid/TestGribGridTimeCS.java (hunk 1 above)
Comment (JohnLCaron) [reply grib/src/test/java/ucar/nc2/grib/grid/TestGribGridTimeCS.java:305]: ill take it out next rev, still debugging that one.

Comment (JohnLCaron) [reply grib/src/test/java/ucar/nc2/grib/grid/TestGribGridTimeCS.java:305]: Actually sorry, that is correct for this test.

**Pull request merged**